lib/perl/Genome/Model/Tools/Validation/ProcessSomaticValidation.pm

```diff
@@ -619,6 +635,11 @@ sub execute {
   my $ref_seq_build = Genome::Model::Build->get($ref_seq_build_id);
   my $ref_seq_fasta = $ref_seq_build->full_consensus_path('fa');
   my $annotation_build_name = $model->annotation_build->name;
+  if(defined $self->reference_transcripts){
+      print STDERR "Model's annotation build overriden. Using " . $self->reference_transcripts . "\n";
```

Comment:
"overridden" is missing a `d`.

Ideally each of these `print STDERR` in this tool would be replaced with `$self->status_message()` or `$self->warning_message()`.  This one could end up something like:
```perl
$self->status_message("Model's annotation build overridden. Using %s", $self->reference_transcripts);
```